app/utils/code-mirror-themes.ts

```diff
@@ -240,6 +241,7 @@ const BASE_STYLE = {
 
       '& .cm-collapsedRangesInner': {
         borderTop: 'none',
+        paddingTop: '6px',
```

Comment:
Where does this magic number come from 🤔 Any way to make this a bit more obvious?